plugins/doc_fragments/k8s_delete_options.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Red Hat | Ansible
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Options for specifying object wait
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+
+    DOCUMENTATION = r'''
+options:
+  delete_options:
+    type: dict
```

Comment:
```suggestion
    type: dict
    version_added: '1.2.0'
```

Reply:
@tima @geerlingguy Can you confirm this?

Reply:
@Akasurde You mean the version of out next release? If so, 1.2.0 is the plan last we discussed.